bentoml/cli/deployment.py

```diff
@@ -233,6 +233,21 @@ def deployment():
     @click.option(
         '--replicas',
         help='Number of replicas. Option applicable to platform: Kubernetes',
+        type=click.INT,
+    )
+    @click.option(
+        '--memory-size',
+        help='Memory size for lambda function. '
```

Comment:
update doc `Maximum Memory Capacity for Lambda Functions, default is 1024 MB, can be set up to 3008 MB.`